lang/java/reef-common/src/main/java/org/apache/reef/client/DriverLauncher.java

```diff
@@ -59,15 +60,21 @@
       .build();
 
   private final REEF reef;
+  private final UserCredentials user;
 
   private LauncherStatus status = LauncherStatus.INIT;
 
   private String jobId;
   private RunningJob theJob;
 
   @Inject
-  private DriverLauncher(final REEF reef) {
+  private DriverLauncher(final REEF reef, final UserCredentials user) {
```

Comment:
Initially, it is just a placeholder, but can be initialized later if we register our application with the RM as Unmanaged AM.